var/spack/repos/builtin/packages/tassel/package.py

```diff
@@ -13,6 +13,7 @@ class Tassel(Package):
     homepage = "https://www.maizegenetics.net/tassel"
     git      = "https://bitbucket.org/tasseladmin/tassel-5-standalone.git"
 
+    version('3.0', git='git://git.code.sf.net/p/tassel/tassel3-standalone.git')
```

Comment:
Can we download a tarball, or fix to a commit sha if we are dealing with a fixed version?